Demos/Blazorise.Demo/Pages/Tests/ListGroupPage.razor

```diff
@@ -75,7 +75,7 @@
             </CardBody>
             <CardBody>
                 <ListGroup Flush="flush">
-                    <ListGroupItem Color="Color.None">None</ListGroupItem>
+                    <ListGroupItem Color="Color.Default">Default</ListGroupItem>
```

Comment:
Just an unrelated idea... could probably just go every available enum dynamically? Easier to stay up to date.

Reply:
Maybe later.